cmd/controller/main.go

```diff
@@ -45,8 +46,11 @@ func main() {
 	flag.IntVar(&options.WebhookPort, "webhook-port", 9443, "The port the webhook endpoint binds to for validation and mutation of resources")
 	flag.IntVar(&options.MetricsPort, "metrics-port", 8080, "The port the metric endpoint binds to for operating metrics about the controller itself")
 	flag.Parse()
-
-	log.Setup(controllerruntimezap.UseDevMode(options.EnableVerboseLogging), controllerruntimezap.ConsoleEncoder())
+	log.Setup(
+		controllerruntimezap.UseDevMode(options.EnableVerboseLogging),
+		controllerruntimezap.ConsoleEncoder(),
+		controllerruntimezap.StacktraceLevel(zapcore.DPanicLevel),
```

Comment:
IIUC, this line should allow us to have another type of annotated error, like `Debug` `Info` and `Panic`.  Is there a case where you see this happening?

Reply:
I made this change because our WARN log lines were dumping annoying and unnecessary (and unfortunately useless) stack traces into our logs. IMO, we only need stack traces on panics.

Reply:
Great!